bayesian/utils.py

```diff
@@ -370,6 +370,85 @@ def get_analyses_from_graph(ecosystem, package, version):
     return resp
 
 
+def get_cves_info(modified_date, ecosystem=None):
+    """Contains Gremlin queries and sub-functions.
+
+    Serves CVEs by date and further filters it by ecosystem if provided.
+    """
+    # Get CVEs information by date
+    cve_nodes_by_date_script_template = """\
+    g.V().has('vertex_label','CVE')\
+    .has('modified_date', modified_date).as('cve')\
+    .in('has_cve').as('epv')\
+    .select('cve','epv').by(valueMap())\
+    dedup()\
+    """
+
+    # Get CVEs by date & ecosystem
+    cve_nodes_by_date_ecosystem_script_template = """\
+    g.V().has('vertex_label','CVE')\
+    .has('modified_date', modified_date)\
+    .has('ecosystem',ecosystem).as('cve')\
+    .in('has_cve').as('epv')\
+    .select('cve','epv').by(valueMap())\
+    dedup()\
+    """
+
+    def get_cves_by_date():
```

Comment:
@cnulenka how are you planning to unit test these inner functions?

Reply:
Should I put all these functions in a class? I shall use object of the class to use all these (inner) functions, instead of a call to the outer function.

Reply:
@cnulenka that'll be a better approach in my view.